instrumentation/restclient/lib/opentelemetry/instrumentation/restclient/patches/request.rb

```diff
@@ -37,13 +47,21 @@ def trace_request # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
             OpenTelemetry::Trace.with_span(span) do
               OpenTelemetry.propagation.inject(processed_headers)
             end
+            safe_execute_hook(config[:request_hook], span, self) unless config[:request_hook].nil?
+
+            span
+          end
+
+          def trace_request # rubocop:disable Metrics/AbcSize
+            span = create_request_span
```

Comment:
Any particular reason to break this out into its own method, besides tidiness? I'd generally prefer to keep it all in `trace_request`, but don't feel super strongly.

Reply:
Yes, it gave me a linter error on a method too many lines

Reply:
OK, sounds good. If you want to change back to being one large method and disable that rule, that's great; we can do in a follow-up PR though.